data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -101,36 +105,65 @@
                         "mm, C, km, count, percentage, category"
                     ]
                 }
-            }
+            },
+            "required": [
+                "dwc:measurementType",
+                "dwc:measurementValue",
+                "dwc:measurementUnit"
+            ]
         },
         "ods:searchFilters": {
             "type": "array",
             "description": "Search filters selected for the virtual collection.",
             "items": {
                 "type": "object",
                 "properties": {
+                    "filterKey": {
+                        "type": "string",
+                        "description": "Key of the filter.",
+                        "examples": [
+                            "country",
+                            "midsLevel"
+                        ]
+                    },
                     "filterValue": {
                         "type": "array",
+                        "items": {
+                            "type": "string"
+                        },
+                        "description": "Value or configuration of the filter.",
                         "examples": [
-                            "topicDiscipline=Zoology&midsLevel=1&country=Indonesia&hasMedia=1"
-                        ],
-                        "description": "Value or configuration of the filter."
+                            ["Netherlands", "Belgium"],
+                            ["1"]
+                        ]
                     }
-                }
-            }
+                },
+                "required": ["filterKey", "filterValue"]
+            },
+            "examples": [
+                [
+                    {
+                        "filterKey": "country",
+                        "filterValue": ["Netherlands", "Belgium"]
+                    },
+                    {
+                        "filterKey": "midsLevel",
+                        "filterValue": ["1"]
+                    }
+                ]
+            ]
         },
         "ltc:basisOfScheme": {
             "type": "string",
             "description": "A summary of the basis or purpose for the LatimerCoreScheme. This property is intended to summarize the reason for grouping a number of ObjectGroups within the LatimerCoreScheme, and the purpose for which the data is intended to be used."
-        },
-        "examples": [
-            "dataset for workshop"
-        ]
+        }
     },
     "required": [
         "rdf:type",
         "ods:id",
         "ods:version",
+        "ods:searchFilters",
+        "dcterms:creator",
         "ltc:ObjectGroup",
         "ltc:collectionName"
```

Comment:
lts:collectionName is not part of the top level object but of the ObjectGroup, if it needs to be required it should be required for that object.

Reply:
ok. good point. ltc:ObjectGroup does not require ltc:collectionName. So I will remove it from here. ltc:ObjectGroup only has one required element ltc:baseTypeOfObjectGroup ("High-level terms describing the fundamental nature of objects in the ObjectGroup.") but this is not relevant for us.